pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -37,6 +39,9 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/dockershim/network"
 	"k8s.io/kubernetes/pkg/util/bandwidth"
 	utilexec "k8s.io/utils/exec"
+
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
+	kubefeatures "k8s.io/kubernetes/pkg/features"
```

Comment:
@AlexeyPerevalov CI is complaining about these:

```
GoCompile: missing strict dependencies:
	/b/f/w/pkg/kubelet/dockershim/network/cni/cni.go: import of "k8s.io/apiserver/pkg/util/feature"
	/b/f/w/pkg/kubelet/dockershim/network/cni/cni.go: import of "k8s.io/kubernetes/pkg/features"
Known dependencies are:
```

perhaps they aren't in the Bazel files?